ui_framework/doc_site/src/components/guide_page/guide_page_container.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
+import { GuidePage } from './guide_page.jsx';
+
+function mapStateToProps(state, ownProps) {
+  return Object.assign({}, ownProps, {
```

Comment:
This is our `reducers/index.js` file: https://github.com/elastic/cloud/blob/master/cloud-ui/public/reducers/index.js

A tad long, but simple, at least. You can for example follow the flow for `getRegion` to understand how it's built up through "gradual knowledge" of the state tree: https://github.com/elastic/cloud/blob/a3557255a4e65cd51897a52640c577262845ee20/cloud-ui/public/reducers/index.js#L115